src/modules/10-common/pages/welcome/WelcomePageV2.tsx

```diff
@@ -132,9 +133,11 @@ export default function WelcomePageV2(): JSX.Element {
                     [licenseStateName]: LICENSE_STATE_VALUES.ACTIVE
                   })
                 }
-                const defaultURL = getModuleToDefaultURLMap(accountId, moduleSelected as Module)[
-                  moduleSelected as Module
-                ]
+                const defaultURL = getModuleToDefaultURLMap(
+                  accountId,
+                  moduleSelected as Module,
+                  props.getStartedVariant as PLG_CD_GET_STARTED_VARIANTS
```

Comment:
nit: Since the prop is optional
```suggestion
                  props?.getStartedVariant as PLG_CD_GET_STARTED_VARIANTS
```